### PR TITLE
Redo explore page

### PR DESCRIPTION
**Pull request by estrattonbailey**
Stacked on #4501 

CleanShot 2024-06-13 at 11 54 03@2x
CleanShot 2024-06-13 at 12 21 39@2x

CleanShot 2024-06-11 at 18 43 29@2x
CleanShot 2024-06-11 at 18 42 45@2x

### CODE REVIEW

File: src/state/queries/feed.ts [hunk 1]
```diff
@@ -299,6 +299,34 @@ export function useSearchPopularFeedsMutation() {
   })
 }
 
+const popularFeedsSearchQueryKeyRoot = 'actor-search'
```
Comment (pfrazee) [inline src/state/queries/feed.ts:302]: I think you copy-pasted the wrong key name

File: src/state/queries/suggested-follows.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ const suggestedFollowsByActorQueryKey = (did: string) => [
   did,
 ]
 
-export function useSuggestedFollowsQuery() {
+export function useSuggestedFollowsQuery(options?: {limit?: number}) {
```
Comment (pfrazee) [inline src/state/queries/suggested-follows.ts:34]: Probably gotta update the query key to include the param

**Comment by pfrazee** [comment]
Hot dog! Got a couple UI thoughts:

CleanShot 2024-06-14 at 09 57 46@2x

On that avi spacing, since we're not gonna get text alignment, I figure why not just give it all more spacing? Could also drop from 4 items to 3 if needed.

CleanShot 2024-06-14 at 10 00 02@2x

I'm really not vibing on the blue border here. I find it really pulls my eye away from the content and doesn't help structure the IA for me.

File: src/state/queries/feed.ts (hunk 1 above)
Comment (estrattonbailey) [reply src/state/queries/feed.ts]: Ah good catch

File: src/state/queries/suggested-follows.ts (hunk 2 above)
Comment (estrattonbailey) [reply src/state/queries/suggested-follows.ts]: Good call

**Comment by estrattonbailey** [comment]
CleanShot 2024-06-14 at 12 28 20@2x

**Review by pfrazee — APPROVED** [review]
Good stuff

**Pull request merged**